packages/amplify-dynamodb-simulator/package.json

```diff
@@ -40,6 +40,10 @@
   },
   "jest": {
     "testEnvironment": "node",
-    "collectCoverage": true
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "src/**/*.{ts,tsx,js,jsx}",
+      "!src/__tests__/"
+    ]
```

Comment:
This package has a non-standard folder structure (no src folder), so this needs to be modified

Reply:
Done, good catch